test/hf20-accounts.test.js

```diff
@@ -20,16 +20,18 @@ describe('steem.hf20-accounts:', () => {
 
   describe('claimAccount', () => {
 
+/*  Skip these tests. Steem-js test infrastructure not set up for testing active auths
+    Blocked by Steem issue #3546
```

Comment:
Is it possible to skip these tests conditionally, based on presence of active key, and/or if config is not testnet? Ideally it would be easy to re-enable any tests and see failures, rather than scan to see which ones have been removed entirely via comment. (This applies to the tests below as well.)

Reply:
The entire key infrastructure for testing needs to be updated. I've documented in issue #458

I believe these tests should be re-enabled and updated as part of that work rather than implementing a temporary conditional system right now.